### ISSUE TITLE
suds no longer supports setuptools consider a switch to suds-py3

### ISSUE THREAD

**Issue by graingert** [psphere-project/psphere#27]

**Comment by WhyNotHugo** [comment]
I'd strongly suggest you consider [`zeep`](https://pypi.org/project/zeep/).

Several years back suds was completely abandoned, and [many forks](https://pypi.org/search/?q=suds&o=) appeared, each fixing different bugs, and only some adding support for python3. It was a bit of a mess, because each fork fixed a different bug from the original `suds`, and only some were python3 compatible. You'd often need two different libraries which each relied on a different fork of suds, and ended up deadlocked.

Eventually, `zeep` seemed to come out as a champion: it included all the fixes from most of the forks out there, and good support for Python3. It also has been actively maintained over the years really well, and has had a lot of improvements.

I maintained one of those forks for a few years too, but I've since killed it pointing users to `zeep` as a better replacement, and recommend others do the same.
